docs/subsystems/widgets.md

```diff
@@ -10,11 +10,8 @@ these features:
 
 - **/ping**
 - **/day** (and /night, /light, /dark)
-- **/poll** (and /todo) (BETA)
-- **zform-enabled messages** for the trivia_quiz bot (BETA)
-
-The beta features are only turned on for chat.zulip.org as
-of this writing.
+- **/poll** (and /todo)
+- **zform-enabled messages** for the trivia_quiz bot
```

Comment:
Is the `zform` system not beta?  I don't remember anything about how it works.

Reply:
(I definitely agree we should remark poll/todo as not beta, thanks for noticing this)

Reply:
I don’t think we need to mark zform as BETA.  It was pretty well tested at the time, and these are basically developer docs.  I don’t feel strongly either way.

Reply:
Yeah, I guess given that this is a developer document, not an end-user production thing, the label probably isn't helpful.